pkgs/top-level/all-packages.nix

```diff
@@ -3755,6 +3755,10 @@ with pkgs;
     buildGoModule = buildGo123Module;
   };
 
+  hellwal = callPackage ./../by-name/he/hellwal/package.nix {
+    stdenv = gccStdenv;
```

Comment:
Does it not work with clang on Darwin? Also, this is not necessary - just directly use gccStdenv within the derivation

Reply:
I'm gonna be honest, it's like this only because it passed nixpkgs-vet - sorry I don't get all of things here yet.

Reply:
So how should it look like?

Reply:
The error really could be improved, sorry for the confusion! :sweat_smile: 

I'll reword: `all-packages.nix` does not need to be changed at all in this PR, the directory `pkgs/by-name/he/hellwal` existing makes the attribute `pkgs.hellwal` exist too. So you can remove this entry here entirely :)

Reply:
Oh, it's fine, thank you for answering and teaching me :D
I did that, should be all good now :)